components/admin_console/admin_definition.jsx

```diff
@@ -1811,6 +1811,16 @@ const AdminDefinition = {
                         help_text_default: 'When true, JavaScript errors are shown in a purple bar at the top of the user interface. Not recommended for use in production.',
                         isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.ENVIRONMENT.DEVELOPER)),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.EnableClientPerformanceDebugging',
+                        label: t('admin.service.performanceDebuggingTitle'),
+                        label_default: 'Enable Client Performance Debugging: ',
+                        help_text: t('admin.service.performanceDebuggingDescription'),
+                        help_text_default: 'When true, users will have access to settings in **Settings > Advanced > Performance Debugging** that will assist in debugging performance issues affecting them. Those settings will only affect them and not other users.',
```

Comment:
When true, users will have access to debugging settings for their account in **Settings > Advanced > Performance Debugging** that will assist in diagnosing performance issues when enabled by the affected users.